src/spikeinterface/preprocessing/filter.py

```diff
@@ -56,6 +57,10 @@ class FilterRecording(BasePreprocessor):
         - numerator/denominator : ("ba")
     ftype : str, default: "butter"
         Filter type for `scipy.signal.iirfilter` e.g. "butter", "cheby1".
+    direction : "forward" | "backward" | "forward-backward", default: "forward-backward"
+        Direction of filtering:
+        - forward and backward filter in just one direction, creating phase shifts in the signal.
```

Comment:
Even though this is very clear it might be worth being super-explicit, for example something like:

```
Direciton of filtering:
    - "forward" - filter is applied to the timeseries in one direction, creating phase shifts
    - "backward" - the timeseries is reversed, the filter is applied and filtered timeseries reversed again. Creates phase shifts in the opposite direction to "forward"
    - "forward-backward" - Applies the filter in the forward and backward direction, resulting in zero-phase filtering. Note this doubles the effective filter order.


```